src/components/MapCard/mapCard.module.scss

```diff
@@ -0,0 +1,77 @@
+$primary-color: #f16b67;
```

Comment:
Colour choices like this should be defined in the global scss files (which is my task, but for now just put it in the global file)